plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -15,35 +33,64 @@
 import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Link;
 import org.eclipse.swt.widgets.Text;
-import org.eclipse.ui.PartInitException;
-import org.eclipse.ui.PlatformUI;
-import org.eclipse.ui.browser.IWorkbenchBrowserSupport;
 import org.eclipse.ui.dialogs.PropertyPage;
 import org.eclipse.ui.forms.widgets.ExpandableComposite;
+import org.eclipse.ui.preferences.ScopedPreferenceStore;
 
-import com.google.cloud.tools.eclipse.ui.util.FontUtil;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class DeployPropertyPage extends PropertyPage {
 
-  private static final String APPENGINE_DASHBOARD_URL = "https://console.cloud.google.com/appengine";
+  public static class VCL implements IValueChangeListener {
+    private String name;
+
+    public VCL(String name) {
+      this.name = name;
+    }
+
+    @Override
+    public void handleValueChange(ValueChangeEvent event) {
+      System.out.println(
+          "VCL[" + name + "]: " + event.diff.getOldValue() + " -> " + event.diff.getNewValue());
+    }
+  }
+
+  private static final String APPENGINE_DASHBOARD_URL =
```

Comment:
This URL also appears in gcloud-eclipse-tools/plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java and may change in the future. Perhaps move to gcloud-eclipse-tools/plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/CloudToolsInfo.java